packages/server/modules/core/authz/loaders/index.ts

```diff
@@ -1,27 +1,31 @@
+import { defineModuleLoaders } from '@/modules/loaders'
 import { getStreamFactory } from '@/modules/core/repositories/streams'
-import { defineLoaders } from '@/modules/loaders'
 import { getFeatureFlags } from '@/modules/shared/helpers/envHelper'
 import { db } from '@/db/knex'
 import { getUserServerRoleFactory } from '@/modules/shared/repositories/acl'
+import { err, ok } from 'true-myth/result'
+import { Authz } from '@speckle/shared'
 
-export const defineModuleLoaders = () => {
+export default defineModuleLoaders(async () => {
```

Comment:
Simplified & improved DX of the way server modules define auth loaders. Previously you had to manually invoke defineModuleLoaders() somewhere and you also had to manually invoke defineLoaders() inside of it and you were not even forced to do it.

Now loaders are dynamically auto-loaded from /<module>/authz/loaders/index.ts and you just have to `export default defineModuleLoaders()` which will return the loaders to auto-load and provide proper TS typing